salt/modules/highstate_doc.py

```diff
@@ -487,16 +486,16 @@ def _blacklist_filter(s, config):
     return False
 
 
-def proccess_lowstates(**kwargs):
+def process_lowstates(**kwargs):
     '''
-    return proccessed lowstate data that was not blacklisted
+    return processed lowstate data that was not blacklisted
 
     render_module_function is used to provide your own.
     defaults to from_lowstate
     '''
     states = []
     config = _get_config(**kwargs)
-    proccesser = config.get('proccesser')
+    processor = config.get('processor')
```

Comment:
This whole section makes me a little nervous. Have all references to this function also been renamed?